CHANGES.rst

```diff
@@ -112,6 +112,10 @@ Release History
   which speed up build times slightly. To re-enable this testing,
   enable the ``fail_fast`` RC setting.
   (`#1532 <https://github.com/nengo/nengo/pull/1532>`__)
+- Higher-order synapses no longer remove unnecessary time-step delays when
```

Comment:
Yes it is. For an example see `test_combined_delay` which failed prior to this PR. The explanation is in https://github.com/nengo/nengo/pull/1535#issuecomment-499200176. To be more accurate could say "Some higher-order synapses" rather than just "Higher-order synapses". Specifically, the ones that have `len(den) - len(num) >= 2` after discretization.

Reply:
Okay cool, thanks!